CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## Version 0.7.3
+
+This is a minor release focusing on greatly improving the
+[.interactive()](https://hvplot.holoviz.org/user_guide/Interactive.html)
+API for flexibly building simple apps using widget controls. Thanks to
+@jbednar and @jlstevens for a few small fixes and many thanks to
+@philippjfr for all the rest!
+
+Features:
+
+- Greatly expanded support for the .interactive() API:
```

Comment:
This appears to be been flattened; all but the very last bullet here should be indented under the first bullet; the whole release is all about .interactive (apart from my one dynspread fix)!

Reply:
I now see your intention here. This is not something we've done before so I've updated it to conform to the previous style with a bit more description on the bullet points to make it clear they are all about `.interactive`.